analyzer/src/funTest/assets/projects/synthetic/gradle-all-dependencies-expected-result-with-curations.yml

```diff
@@ -64,6 +64,21 @@ projects:
         dependencies: []
```

Comment:
Commit message nits:

- Drop "the" before "scopes" in the second line.
- Drop "in the library project" in the second-to-last line.

Reply:
Done.